src/utils/getCookie.ts

```diff
@@ -0,0 +1,17 @@
+const getCookie = cname => {
```

Comment:
Let's make this a real function.

Reply:
@domoritz Can you elaborate? I don't get you. :/

Reply:
If you can, use `function` instead of making this a `const gteCookie`.